ape_safe/accounts.py

```diff
@@ -323,8 +390,35 @@ def _contract_approvals(self, safe_tx: SafeTx) -> Dict[AddressType, MessageSigna
         }
 
     def _all_approvals(self, safe_tx: SafeTx) -> Dict[AddressType, MessageSignature]:
-        # TODO: Combine with approvals from SafeAPI
-        return self._contract_approvals(safe_tx)
+        approvals = self.get_api_confirmations(safe_tx)
+        # NOTE: Do this last because it should take precedence
+        approvals.update(self._contract_approvals(safe_tx))
+        return approvals
+
+    def submit_safe_tx(
+        self,
+        safe_tx: SafeTx,
+        submitter: Union[AccountAPI, AddressType, str, None] = None,
+        **txn_options,
+    ) -> ReceiptAPI:
+        """
+        Submit the safe transaction using the submitter after all signatures have been collected.
+
+        Args:
+            safe_tx (``SafeTX``): The safe transaction to submit.
+            submitter (Union[``AccountAPI``, ``AddressType``, str, ``None``]):
+                The submitter to use for the transaction. Defaults to ``None``.
+
+        Returns:
+            :class:`~ape.api.transactions.ReceiptAPI`
```

Comment:
Sadly, this link won't work.
You might as well just do:

```suggestion
            ``ReceiptAPI``
```

for anything in Ape